Assets/Scripts/AltCamera.cs

```diff
@@ -21,6 +22,6 @@ void Start()
     }
 
     void Update() {
-        _thisCamera.fieldOfView = _copiedCamera.fieldOfView;
+        if (_isCopyingFOV) _thisCamera.fieldOfView = _copiedCamera.fieldOfView;
```

Comment:
Similar to my comment on how we change isWatchShown, I think it may be better to make a public function for this that gets called whenever copiedCamera's fov changes, instead of always doing it in Update. But tbh I don't think this is as important here

Reply:
The issue is that whenever the main camera changes FOV, it is a transition and not instant. So then I would have to write some complex coroutine that copies the main camera's FOV each frame until the main camera stops moving.

I think it is fine to just keep it as is for now if there are no visible performance issues.